scripts/init.sh

```diff
@@ -42,6 +42,35 @@ start-parachain)
     --rpc-cors all \
     --ws-external \
     --rpc-methods=Unsafe \
+    --state-cache-size 0 \
+    --log="main,debug" \
+  ;;
+
+start-parachain-sibling)
```

Comment:
The only reason to have this duplicated section is due to the ports, instead of duplicating I would let pass 3 ports, or even better add the para_id to the base ports. Like 30333 + 2000 or whatever paraid is passed.

Reply:
Great suggestion, done 👍 Thank you!